components/ai_chat/core/browser/ai_chat_service.cc

```diff
@@ -426,6 +426,11 @@ void AIChatService::OpenConversationWithStagedEntries(
       associated_content->GetContentId(), associated_content);
   CHECK(conversation);
 
+#if !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
+  if (ai_chat_metrics_ != nullptr) {
+    ai_chat_metrics_->HandleOpenViaEntryPoint(EntryPoint::kBraveSearch);
+  }
```

Comment:
Maybe update unittest of AIChatService::OpenConversationWithStagedEntries to include this too?

Reply:
We have coverage in `AIChatMetricsUnitTest`, along with the other entry points

Reply:
Discussed in the DM, I'm okay with it.